libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboardStyles.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  IStyle,
+  mergeStyleSets,
+  IProcessedStyleSet,
+  getTheme
+} from "@fluentui/react";
+export interface IVisionExplanationDashboardStyles {
+  cohortDropdown: IStyle;
+  cohortPickerLabel: IStyle;
+  cohortPickerLabelWrapper: IStyle;
+  line: IStyle;
+  page: IStyle;
+  searchBox: IStyle;
+  filterButton: IStyle;
+  toolBarContainer: IStyle;
+  mainContainer: IStyle;
+  halfContainer: IStyle;
+  imageListContainer: IStyle;
+  slider: IStyle;
+  tabs: IStyle;
+}
+
+export const visionExplanationDashboardStyles: () => IProcessedStyleSet<IVisionExplanationDashboardStyles> =
+  () => {
+    const theme = getTheme();
+    return mergeStyleSets<IVisionExplanationDashboardStyles>({
+      cohortDropdown: {
+        width: "300px"
+      },
+      cohortPickerLabel: {
+        fontWeight: "600"
+      },
+      cohortPickerLabelWrapper: {
+        paddingBottom: "5px"
+      },
+      filterButton: {
+        height: "32px"
+      },
+      halfContainer: {
+        height: "100%",
+        width: "50%"
+      },
+      imageListContainer: {
+        border: "1px solid #D2D4D6",
+        height: "100%",
+        overflow: "scroll"
+      },
+      line: {
+        borderTop: "1px solid #EDEBE9"
+      },
+      mainContainer: {
+        alignItems: "center",
+        justifyContent: "space-between",
+        width: "100%"
+      },
+      page: {
+        color: theme.semanticColors.bodyText,
+        height: "100%",
+        padding: "0 40px 10px 40px",
+        width: "100%"
+      },
+      searchBox: {
+        width: "300px"
+      },
+      slider: {
+        width: "320px"
+      },
+      tabs: {
+        display: "flex",
```

Comment:
can you use stack to remove flex here?

Reply:
The Fluent UI doc says that Pivot shouldn't have any children that aren't Pivot Item. Should I put a stack in there anyways?